ci_scripts/ci_post_clone.sh

```diff
@@ -12,32 +12,29 @@ cd /Volumes/workspace/repository
 git submodule init
 git submodule update
 
+# add necessary directories to path
+export PATH=/Users/local/.gem/ruby/2.6.0/bin:/Users/local/Library/Python/3.8/bin:$PATH
+
 if [ $CI_PRODUCT_PLATFORM == 'macOS' ]
 then
   # generate qt_static_macos
   auth_header="$(git config --local --get http.https://github.com/.extraheader)"
   git clone https://github.com/mozilla-mobile/qt_static_macos
   cd qt_static_macos
-  cat x* > qt_static.tar.gz
+  cat qt6* > qt_static.tar.gz
   tar xf qt_static.tar.gz
   cd ..
-  export QT_MACOS_BIN=`pwd`/qt_static_macos/qt/bin
-  export PATH=`pwd`/qt_static_macos/qt/bin:$PATH
+  export QT_MACOS_BIN=`pwd`/qt_static_macos/qt6/bin
+  export PATH=`pwd`/qt_static_macos/qt6/bin:$PATH
 else
-  # generate qt_ios
-  git clone https://github.com/mozilla-mobile/qt_ios
```

Comment:
Can we remove the qt_ios repo?

Reply:
Currently we are still using it for the github ci but I can work on that as well.